pkg/kubelet/kubelet.go

```diff
@@ -974,6 +976,42 @@ func (kl *Kubelet) syncNodeStatus() {
 	}
 }
 
+// relabelVolumes relabels SELinux volumes to match the pod's
+// SELinuxOptions specification. This is only needed if the pod uses
+// hostPID or hostIPC. Otherwise relabeling is delegated to docker.
```

Comment:
@thockin The chcon runner can definitely be re-used to provide support for rkt and other runtimes.  I'll make a follow-up issue.